### ISSUE TITLE
Bleeding content on iPad/smaller devices

### ISSUE THREAD

**Issue by NathanInnes** [flybywiresim/website#93]
Community section statistics bleed of page
![image](https://user-images.githubusercontent.com/18389085/107870005-9806ee80-6e8c-11eb-92c1-7fa83b76ddf4.png)

Experienced on iPad Air

**Comment by Benjozork** [comment]
Also what is wrong with that title font...

**Comment by NathanInnes** [comment]
Most likely 
```
html {
  font-family: -apple-system, BlinkMacSystemFont, 'Nunito Sans', 'Segoe UI', 'Roboto', 'Oxygen',
  'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
  font-size: 14px;
  letter-spacing: .3px;
  scroll-behavior: smooth;
}
```

Located in index.css causing the font issue.

**Comment by NathanInnes** [comment]
Fixed in 664fc2c8f0239d0ab0af610411a63f0f5ea092ee
